specs/commandline/key.md

```diff
@@ -115,10 +115,33 @@ notation key list
 
 Upon successful execution, a list of keys is printed out with information of name, key path, certificate path, key id and plugin name. The default signing key name is preceded by an asterisk. The key id and plugin name are used together to provide the information of the key identifier for the remote key and the plugin associated with it.
 
-### Delete two keys from signing key list
+### Remove a specified key from Notation signing key list
+
+```shell
+notation key delete <key_name>
+```
+
+- Upon successful removal of a local testing key created by notation, the output message is printed out as follows:
+
+```text
+Removed <key_name> from Notation signing key list. The source key file still exists.
+```
+- Upon successful removal of a key associated with a KMS, the output message is printed out as follows:
+
+```text
+Removed <key_name> from Notation signing key list. The source key files still exist.
```

Comment:
If the source key is a remote key, it may not be a file. How about "The source key still exists."  Maybe we can use this for local key as well.